### PR TITLE
Transplant blurb on the choice of the name "Official Images"

### PR DESCRIPTION
**Pull request by wglambert**
From https://github.com/docker-library/official-images/pull/9379#discussion_r552293090

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ As stewards of the official images and maintainers of many images ourselves, we
 
 ### What do you mean by "Official"?
 
-This question is so common it's answered in our primary repository!
+The name of this program was chosen in an attempt to reflect that upstream-first focus (although in hindsight, it's clear that it was a choice with some amount of confusion potential for which we're sorry). 
 
-See [the answer in the readme of the `github.com/docker-library/official-images` repository](https://github.com/docker-library/official-images#what-do-you-mean-by-official).
+This question is so common it's answered in our primary repository! See [the answer in the readme of the `github.com/docker-library/official-images` repository](https://github.com/docker-library/official-images#what-do-you-mean-by-official).
```
Comment (tianon) [inline README.md:37]: Given we're ditching this section in that readme, we should probably reword this a bit:

```suggestion
See [the readme of the `github.com/docker-library/official-images` repository](https://github.com/docker-library/official-images#what-are-official-images) for a more verbose overview of the program.
```

File: README.md [hunk 2]
```diff
@@ -32,9 +32,9 @@ As stewards of the official images and maintainers of many images ourselves, we
 
 ### What do you mean by "Official"?
 
-This question is so common it's answered in our primary repository!
+The name of this program was chosen in an attempt to reflect that upstream-first focus (although in hindsight, it's clear that it was a choice with some amount of confusion potential for which we're sorry).
```
Comment (tianon) [inline README.md:35]:
```suggestion
The name of this program was chosen in an attempt to reflect our upstream-first focus (although in hindsight, it's clear that it was a choice with some amount of confusion potential for which we're sorry). 
```

File: README.md (hunk 1 above)
Comment (yosifkit) [inline README.md:37]: The link to "what-do-you-mean-by-official" will no longer work after https://github.com/docker-library/official-images/pull/9379.

**Review by tianon — APPROVED** [review]
:metal:

**Pull request merged**